src/wp-includes/block-template-utils.php

```diff
@@ -38,20 +38,8 @@
  */
 function get_block_theme_folders( $theme_stylesheet = null ) {
 	$theme_name = null === $theme_stylesheet ? get_stylesheet() : $theme_stylesheet;
```

Comment:
```suggestion
function get_block_theme_folders( $theme_stylesheet = '' ) {
```

This is handled here 

https://github.com/WordPress/wordpress-develop/blob/60d2d8ac75c4350a45e4c6970c7d0295b757f1f0/src/wp-includes/theme.php#L117-L122